src/Advection/upwind_biased_advective_fluxes.jl

```diff
@@ -17,83 +17,83 @@ const UpwindScheme = AbstractUpwindBiasedAdvectionScheme
 
 @inline function advective_momentum_flux_Uu(i, j, k, grid, scheme::UpwindScheme, U, u)
 
-    ũ  =    _symmetric_interpolate_xᶜᵃᵃ(i, j, k, grid, scheme, U)
+    ũ  =    _symmetric_interpolate_xᶜᵃᵃ(i, j, k, grid, scheme, Ax_qᶠᶜᶜ, U)
     uᴸ =  _left_biased_interpolate_xᶜᵃᵃ(i, j, k, grid, scheme, u)
     uᴿ = _right_biased_interpolate_xᶜᵃᵃ(i, j, k, grid, scheme, u)
 
-    return Axᶜᶜᶜ(i, j, k, grid) * upwind_biased_product(ũ, uᴸ, uᴿ)
+    return upwind_biased_product(ũ, uᴸ, uᴿ)
```

Comment:
This isn't a flux if we don't multiply by area, right? Where do we multiply by area?